.kokoro/tests/run_tests.sh

```diff
@@ -92,8 +100,27 @@ for file in **/requirements.txt; do
     file=$(dirname "$file")
     cd "$file"
 
+    # First we look up the environment variable `RUN_TESTS_DIRS`. If
+    # the value is set, we'll iterate through the colon separated
+    # directory list.
+    if [[ -n "${RUN_TESTS_DIRS:-}" ]]; then
```

Comment:
The other variables are actually set by a flag at the start of this script. Should we do the same for this one (or document?

Reply:
Added some documents